common-test-lib/src/main/java/com/google/cloud/tools/intellij/testing/apis/TestCloudLibrary.java

```diff
@@ -100,6 +105,8 @@ public static TestCloudLibraryClient createEmpty() {
       return create("", "", "", "", "", "", TestCloudLibraryClientMavenCoordinates.createEmpty());
```

Comment:
It looks like think this is where spotlessJava check fails - we probably need to reformat spaces in this file.